modules/users/shiv.tf

```diff
@@ -0,0 +1,12 @@
+data "discord_member" "shiv" {
+  server_id = data.discord_server.server.id
+  user_id   = 998314422055489596
+}
+
+resource "discord_member_roles" "shiv" {
+  server_id = data.discord_server.server.id
+  user_id   = data.discord_member.arvindpunk.id
```

Comment:
can you change the username from arvindpunk -> shiv

Reply:
Ok i will change it